### PR TITLE
feat: P4PU-171 dr auth oneidentity

### PR DESCRIPTION
**Pull request by oleksiybozhykntt**
#### List of Changes
added sequnce diagram for DR auth oneidentity

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: design/arc/oneidentity-integration/sequence_diagram_authentication.mmd
Comment (and-mora) [inline design/arc/oneidentity-integration/sequence_diagram_authentication.mmd:1]: what's the purpose of all the `+` in each link?

Comment (oleksiybozhykntt) [reply design/arc/oneidentity-integration/sequence_diagram_authentication.mmd:1]: removed the activation sign from sequnce

Comment (and-mora) [reply design/arc/oneidentity-integration/sequence_diagram_authentication.mmd:1]: next moves to improve the diagrams are: dotted line for responses with `-->>` and activation blocks with `->>+` `->>-` to highlight a blocking process

Comment (oleksiybozhykntt) [reply design/arc/oneidentity-integration/sequence_diagram_authentication.mmd:1]: added requested improvements into sequnce

File: design/arc/oneidentity-integration/sequence_diagram_system_characteristics.mmd [hunk 1]
```diff
@@ -0,0 +1,16 @@
+sequenceDiagram
+    participant User-Agent
+    participant ARC
+    participant OneIdentity as One Identity
+
+    User-Agent->>ARC: Richiesta accesso all'applicazione
```
Comment (and-mora) [inline design/arc/oneidentity-integration/sequence_diagram_system_characteristics.mmd]: I don't understand this action, is it a login action or access to protected resources?

File: design/arc/oneidentity-integration/sequnce_diagram_token.mmd
Comment (and-mora) [inline design/arc/oneidentity-integration/sequnce_diagram_token.mmd:1]: typo in filename

File: design/arc/oneidentity-integration/sequence_diagram_system_characteristics.mmd (hunk 1 above)
Comment (oleksiybozhykntt) [reply design/arc/oneidentity-integration/sequence_diagram_system_characteristics.mmd]: fixed the missunderstanding

File: design/arc/oneidentity-integration/sequnce_diagram_token.mmd
Comment (oleksiybozhykntt) [reply design/arc/oneidentity-integration/sequnce_diagram_token.mmd:1]: renamed

**Review by and-mora — APPROVED** [review]

**Pull request merged**